### DISCUSSION TITLE
Application Error when adding only loader route

### DISCUSSION THREAD

**Discussion by JakovGlavac** [remix-run/remix#1660] [Q&A] [answered]
Hi, I'm trying to add sitemap.xml to my remix site, but when I add route that only has loader function I get an error like this:
![image](https://user-images.githubusercontent.com/34971091/151138837-36ef3d40-bef8-4f03-82c2-c2e91e98376f.png).

I tried using example from [remix.run](https://remix.run/docs/en/v1/tutorials/jokes#resource-routes) and example from [dev.to](https://dev.to/chrisbenjamin/tutorial-add-sitemapxml-and-robotstxt-to-remix-site-4n23), but still no luck.

I really don't know what to do.

**Comment by machour** [comment]
@JakovGlavac if this is still a problem, can you provide a reproduce case? (either via codesandbox or a repo)

**Comment by JakovGlavac** [comment, marked as answer]
Yeah I had some stupid Prisma error, but it wasn't obvious from shown error.
In the future it would be nice to have better error messages.
